samples/grpc/shopping-cart-service-scala/src/main/resources/cluster.conf

```diff
@@ -22,11 +22,6 @@ akka.management {
     port = 8558
     port = ${?HTTP_MGMT_PORT}
   }
-  cluster.bootstrap {
-    contact-point-discovery {
-      discovery-method = kubernetes-api
-      required-contact-point-nr = 1
-      required-contact-point-nr = ${?REQUIRED_CONTACT_POINT_NR}
-    }
-  }
+  # actual bootstrap/discovery config would go here
+  # for running locally see local-shared.conf
```

Comment:
why can't we have this included?

Reply:
I dropped the kubernetes yamls in a previous PR so it doesn't really make sense to include kubernetes discovery config?

Reply:
ok